electronics/ohms_law.py

```diff
@@ -0,0 +1,32 @@
+# Importing doctest for testing our function
+from doctest import testmod
+
+# Defining function
+
+
+def ohms_law(voltage: float, current: float, resistance: float) -> float:
```

Comment:
```suggestion
def ohms_law(voltage: float = 0, current: float = 0, resistance: float = 0) -> float:
```
Default Values be 0 (It will not affect the solution but is recommended by me)

Reply:
Yes i did it in previous commit but the max line length was exceeded by this because the parameters name also very long then the styling spaces , i just try again hope it should work, thank you so much for your interest.

Reply:
Hello, You can commit my changes and then run the following commands in root directory of repository. I assume you are making changes to a local copy of repository.

* `pip install pre-commit`
* `pre-commit install`
* `pre-commit run -a`

Then upload the changes